.github/workflows/gh-pages.yml

```diff
@@ -32,7 +32,13 @@ jobs:
           python -m pip install --upgrade pip
           python -m pip install -r requirements.txt -r requirements-dev.txt
           python -m pip install '.[all]'
+      - uses: actions/cache@v4
+        id: cache-docs
+        with:
+          path: docs/_examples_database/
+          key: ${{ runner.os }}-sphinx
```

Comment:
to do: need to add `restore` and `save` actions 

https://github.com/actions/cache?tab=readme-ov-file#using-a-combination-of-restore-and-save-actions